assets/docs/rishit-dagli/swin-transformers/1.md

```diff
@@ -0,0 +1,35 @@
+# Module rishit-dagli/swin-transformers/1
+
+Swin Transformer model, a general purpose backbone for computer vision trained on CIFAR-100.
+
+<!-- task: image-classification -->
+<!-- network-architecture: other -->
```

Comment:
Since the Swin Transformer is described as "can be a general-purpose backbone for computer vision", do you think it would make sense to change this to
```
<!-- network-architecture: swin-transformer -->
```
and add a new entry in tags/network_architecture.yaml? I think that would be useful in case someone publishes a variant that was trained on imagenet instead of cifar etc.

Reply:
Sure thing, made that change.